src/foam/nanos/dig/DIG.js

```diff
@@ -53,6 +53,18 @@ foam.CLASS({
     },
     'cmd',
     'format',
+    {
+      name: 'digColumns',
+      expression: function(daoKey) {
+         var dao = this.__context__[daoKey];
+
+         if ( dao.of.getAxiomByName('digColumns') ) {
+          return dao.of.getAxiomByName('digColumns').columns.toString();
+        } else {
+          return dao.of.getAxiomsByClass(foam.core.Property).toString();
```

Comment:
this isn't right. You want an array of strings, not the string of an array.
`return dao.of.getAxiomsByClass(foam.core.Property).map(p => p.name);`